pkgs/development/python-modules/python-coinmarketcap/default.nix

```
@@ -0,0 +1,35 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  pytestCheckHook,
+  requests,
+}:
+
+buildPythonPackage rec {
+  pname = "python-coinmarketcap";
+  version = "0.6";
+  format = "setuptools";
```

```suggestion
  pyproject = true;
```
`format` is a legacy nixpkgs system